dGame/dComponents/SimplePhysicsComponent.h

```diff
@@ -14,16 +14,24 @@
 
 class Entity;
 
+enum eClimbableType : int32_t {
+	CLIMBABLE_TYPE_NOT,
```

Comment:
This should specify zero as the starting number
`climbable_type_not = 0,
other values`